src/js/view/controls/components/menu/menu.js

```diff
@@ -123,6 +124,12 @@ export default class Menu extends Events {
         prependChild(this.mainMenu.topbar.el, backButton.element());
         return backButton;
     }
+    setBackButtonAriaLabel(labelText) {
```

Comment:
Screen readers will always say Button at the end if the aria-role is button. We should instead set the inner text and the aria label to be `labelText` if the argument is passed in, with localization.prev as the fallback value.

Reply:
Great call. Testing with Voiceover now & will update accordingly.

Reply:
Okay, now it says something like "Previous - Subtitle Settings, button" which seems right!